Plan/src/main/java/com/djrapitops/plan/system/database/databases/sql/tables/TransferTable.java

```diff
@@ -51,15 +51,33 @@ public TransferTable(SQLDB db) {
         super(TABLE_NAME, db);
 
         serverTable = db.getServerTable();
-        insertStatementNoParts = "REPLACE INTO " + tableName + " (" +
-                Col.SENDER_ID + ", " +
-                Col.EXPIRY + ", " +
-                Col.INFO_TYPE + ", " +
-                Col.EXTRA_VARIABLES + ", " +
-                Col.CONTENT +
-                ") VALUES (" +
-                serverTable.statementSelectServerID + ", " +
-                "?, ?, ?, ?)";
+
+        if (db.isUsingH2()) {
+            insertStatementNoParts = "INSERT INTO " + tableName + " (" +
```

Comment:
Does H2 not support `REPLACE INTO`? - That might also in use elsewhere in the queries

Reply:
REPLACE INTO / MERGE INTO throws an error for not finding the PRIMARY KEY
This is the only place in the code REPLACE INTO is used.